src/Microsoft.Azure.ServiceBus/MessageSession.cs

```diff
@@ -21,6 +21,24 @@ public MessageSession(string sessionId, DateTime lockedUntilUtc, MessageReceiver
             this.ReceiveMode = innerMessageReceiver.ReceiveMode;
         }
 
+        /// <summary>
+        /// Gets the DateTime that the current receiver is locked until. This is only applicable when Sessions are used.
```

Comment:
>This is only applicable when Sessions are used. [](start = 73, length = 47)

this is description on sessions in the MessageSession file, so we can skip this part